pom.xml

```diff
@@ -100,6 +100,20 @@ SOFTWARE.
       <artifactId>commons-cli</artifactId>
       <version>1.4</version>
     </dependency>
+    <dependency>
+      <groupId>com.jcabi</groupId>
+      <artifactId>jcabi-xml</artifactId>
+      <version>0.21.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.llorllale</groupId>
+      <artifactId>cactoos-matchers</artifactId>
+      <version>0.18</version>
+    </dependency>
```

Comment:
@olenagerasimova why do we need these dependencies for `compile` scope? I think it'd be enough to use `test` scope here, also `hamcrest` dep provided by parent pom with `test` scope.